pkgs/development/misc/rpiboot/unstable.nix

```diff
@@ -26,11 +24,11 @@ in stdenv.mkDerivation {
     cp -r msd $out/share/rpiboot
   '';
 
-  meta = {
+  meta = with stdenv.lib; {
     homepage = "https://github.com/raspberrypi/usbboot";
     description = "Utility to boot a Raspberry Pi CM/CM3/Zero over USB";
-    maintainers = [ stdenv.lib.maintainers.cartr ];
-    license = stdenv.lib.licenses.asl20;
-    platforms = stdenv.lib.platforms.unix;
+    license = licenses.asl20;
+    maintainers = with maintainers; [ cartr ];
+    platforms = [ "aarch64-linux" "armv7l-linux" "armv6l-linux" ];
```

Comment:
This technically builds on regular Linux, but it's really targeted at ARM devices/the Pi, so I think this is reasonable.
https://github.com/raspberrypi/usbboot/blob/d3760e119385a179765f43a50a8e051a44127c25/Readme.md#building

Reply:
Isn't this a tool I could run on my workstation to "fastload" a raspberry Pi? If so, building it on `x86_64-linux` too would be appreciated ;-)